overlord/snapstate/snapstate_update_test.go

```diff
@@ -6837,6 +6837,104 @@ func (s *snapmgrTestSuite) TestUpdatePrerequisiteWithSameDeviceContext(c *C) {
 	})
 }
 
+/*
+func (s *validationSetsSuite) TestUpdateManyValidationSetsPartialFailure(c *C) {
```

Comment:
what's the plan/status with this?

Reply:
Yes, sorry, I missed this when re-basing and re-working this in a new branch. Uncommented and fleshed out (was related to the untested bits you pointed above).